acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -46,6 +46,22 @@
     assert(/^#{gem_name}/.match(stdout), "#{gem_name} was not found after installation.")
   end
 
+  if gem_name == 'excon'
+    step "SERVER-1601: Validate use of excon gem successful as puppet user" do
+      if options[:type] == 'pe'
+        runuser = 'pe-puppet'
+      else
+        runuser = 'puppet'
+      end
+      on(master, "su #{runuser} -s /bin/bash -c "\
+        "'/opt/puppetlabs/bin/puppetserver ruby "\
+        "-rexcon -e \"puts Excon::VERSION\"'") do
+        assert_equal(gems['excon'], stdout.strip,
+                     "Unexpected output for excon version")
+      end
+    end
+  end
```

Comment:
Yes, it would.  You'd see an exception/failure to load the "excon" gem.  Thought this test would be useful for future regression testing along with the fix.